src/main/java/org/prebid/cache/metrics/MetricsRecorder.java

```diff
@@ -14,7 +14,8 @@
         REQUEST_INVALID("pbc.request.invalid"),
         JSON("pbc.${prefix}.json"),
         XML("pbc.${prefix}.xml"),
-        ERROR_DB("pbc.${prefix}.err.db");
+        ERROR_DB("pbc.${prefix}.err.db"),
+        SECONDARY_CACHE_WRITE_ERROR("error.secondary.cache.write");
```

Comment:
This one should follow convention for enum values and metric names: `ERROR_SECONDARY_WRITE("pbc.err.secondaryWrite")` or alike